pkg/repositories/gormimpl/project_repo.go

```diff
@@ -70,3 +70,15 @@ func NewProjectRepo(db *gorm.DB, errorTransformer errors.ErrorTransformer,
 		metrics:          metrics,
 	}
 }
+
+func (r *ProjectRepo) UpdateProject(ctx context.Context, projectUpdate models.Project) error {
+	// Use gorm client to update the two fields that are changed.
+	writeTx := r.db.Model(&projectUpdate).Update("Description", "Labels")
```

Comment:
i think this is not quite right (see http://gorm.io/docs/update.html#Update-Changed-Fields)
this translates to //// UPDATE projects SET Description='Labels'

can you do something similar to https://github.com/lyft/flyteadmin/blob/master/pkg/repositories/gormimpl/launch_plan_repo.go#L43